spacewalk/certs-tools/spacewalk-certs-tools.changes.dfaltum.master

```diff
@@ -0,0 +1 @@
+- improve the parsing of the authorityKeyIdentifier
```

Comment:
```suggestion
- fix parsing Authority Key Identifier when keyid is not prefixed (bsc#1229079)
```

This change is more for the internals as we had also a bugreport. So we must add that bugnumber.